frontend/src/components/viewers/MarkdownViewer.test.tsx

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import { mount, ReactWrapper } from 'enzyme';
+import MarkdownViewer, { MarkdownViewerConfig } from './MarkdownViewer';
+import { PlotType } from './Viewer';
+
+describe('MarkdownViewer', () => {
+  let tree: ReactWrapper<any> | null;
+
+  afterEach(() => {
+    if (tree) {
```

Comment:
_**[nit]** Could you perhaps do an early return to prevent unnecessary code depth, like:_
```js
if (!tree) return;
tree.unmount();
tree = null;
```

Reply:
Sure, done.